src/components/VAction/VAction.vue

```diff
@@ -32,6 +32,14 @@ export default {
     if (tag === 'RouterLink') {
       options.nativeOn = listeners;
     }
+    /**
+     * Supress Vue warnings of .native modifiers
+     * Reference comment and issue #9999 on Vuetify
+     * https://github.com/vuetifyjs/vuetify/issues/9999#issuecomment-579434865
+     */
+    if (tag === 'button') {
+      options.nativeOn = null;
+    }
```

Comment:
Do we need this? Isn't nativeOn already undefined for a button? It's only added to `RouterLink` above

Reply:
You are correct that it is undefined on the button. However Vue has a nasty annoying `[warn]` label `error` that is only alleviated by explicitly removing it. It just makes for just a bit of nicer DX and doesn't affect the overall functionality of the component.

I reference this:
`Supress Vue warnings of .native modifiers`
     * Reference comment and issue #9999 on Vuetify
     * [Issue #9999-Comment](https://github.com/vuetifyjs/vuetify/issues/9999#issuecomment-579434865)